tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+	"extends": "./tsconfig.build.json",
```

Comment:
Should this just reference `./packages/blueprints/tsconfig.build.json`? The actual `./tsconfig.build.json` seems to be incorrect anyway, since it has `baseUrl` as `./`?